demos/collection/lister-ipp.php

```diff
@@ -50,7 +50,7 @@
     $row->iso = mb_strtolower($row->iso);
 });
 $lister->setModel(new Country($app->db))
-    ->addCondition('id', -1); // no such records so model will be empty
+    ->addCondition(Country::hinting()->fieldName()->id, -1); // no such records so model will be empty
```

Comment:
Why not `(new Country())->id_field)` ?

Reply:
also possible, but this seems better to me when we expect hintable `id` is id field

Reply:
best would be `addCondition(false)`, right? :)

Reply:
Or `addCondition(1, '<>', 1)` :)